web/index.html

```diff
@@ -32,6 +32,15 @@
                 left: 2px;
                 bottom: 2px;
                 z-index: 99999;
+				display: flex;
+            }
+		
+			.info-col {
```

Comment:
You are mixing spaces and tabs here. Please keep the indentation style consistent.